installscripts/cookbooks/jenkins/attributes/default.rb

```diff
@@ -13,7 +13,9 @@
 default['authfile']="/home/#{node['jenkins']['SSH_user']}/cookbooks/jenkins/files/default/authfile"
 default['jenkins']['scriptApprovalfile']="/home/#{node['jenkins']['SSH_user']}/cookbooks/jenkins/files/scriptapproval/scriptApproval.xml"
 default['jenkins']['scriptApprovalfiletarget']="#{node['jenkins']['home']}/scriptApproval.xml"
-default['bitbucketelb']='jazz-bitbucketelb.us-east-1.elb.amazonaws.com'
+default['scm']='bitbucket'
+default['scmelb']='jazz-bitbucketelb.us-east-1.elb.amazonaws.com'
```

Comment:
This is incorrect in this merge, we cannot bring back things that are removed in the release